scripts/coccinelle/test-operator-cleanup

```diff
@@ -1,4 +1,17 @@
 #!/usr/bin/perl -w -p -i
+#
+# Copyright (c) 2001 Matej Pfajfar.
+# Copyright (c) 2001-2004, Roger Dingledine.
+# Copyright (c) 2004-2006, Roger Dingledine, Nick Mathewson.
+# Copyright (c) 2007-2019, The Tor Project, Inc.
+# See LICENSE for licensing information
+
+# This script looks for instances of C comparison operators as macro arguments,
+# and replaces them with our OP_* equivalents.
+#
+# Some macros that take operators are our tt_int_op() testing macro, and the
+# standard timercmp() macor.  Coccinelle can't handle their syntax, however,
```

Comment:
```suggestion
# standard timercmp() macro.  Coccinelle can't handle their syntax, however,
```

Reply:
Fixup is e45fee1fefa6578375f2a83b8ba4ac66ea7c4e15